site/app/templates/officeHoursQueue/QueueButtonBar.twig

```diff
@@ -11,6 +11,9 @@
   <button id="notification_btn" class="btn btn-default filter_btn" type="button" onclick="enableNotifications()">
     Enable Notifications
   </button>
+  <a id="view_stats" class="btn btn-default filter_btn" type="button" href="{{base_url}}/stats">
```

Comment:
HTML/CSS IDs should use dashes not underscores

https://submitty.org/developer/coding_style_guide/css